api_v3/lib/types/entry/KalturaBaseEntry.php

```diff
@@ -345,7 +345,16 @@ class KalturaBaseEntry extends KalturaObject implements IRelatedFilterable
 	 * @var string
 	 * @filter matchand
 	 */
-	public $entitledUsersPublish;	
+	public $entitledUsersPublish;
+
+	/**
+	 * Comma seperated string of the capabilities of the entry, each plugin can add it's own capability
```

Comment:
I think it's not necessarily used by plugins, please make the attribute description more generic.